tasks/drupal.yml

```diff
@@ -44,15 +44,15 @@ tasks:
       - echo "🚮 Dropping existing database"
       - ./vendor/bin/drush {{ .site }} sql:drop --yes
       - echo "📰 Importing database"
-      - ./vendor/bin/drush {{ .site }} sql:query --file=$DB_DIR/db.sql.gz
+      - gunzip < $DB_DIR/db.sql.gz | ./vendor/bin/drush {{ .site }} sql:cli
```

Comment:
A few years ago @m4olivei suggested using pigz instead of gunzip because it had significant performance benefits. 

https://www.clouvider.com/knowledge_base/compression-options-in-linux-gzip-vs-pigz/

Reply:
Ohh, thanks for linking that article. I never did learn the reason _why_ `pigz` was faster, just that it was 💡

Reply:
I don't think pigz is in the default ddev image, so we'd need to add a Dockerfile to install it. Which IMO is fine, we do it on most of our projects anyways.

I do wonder if it's worth asking about adding it to the default web images in ddev upstream.

Reply:
@mrdavidburns 
I tried `pigz` with the following command:
* `pigz -dc file.gz`

But the time execution was more than 2 minutes, so I unzipped the file before importing with `sql:query` and it worked fine, it takes the same time execution with `pigz` and `gunzip` so I left `gunzip` because the other needed to add a Dockerfile to install it as @deviantintegral mentioned

Reply:
@alexanderpatriciop Were you trying pigz with an existing project or just a fresh site install?

Reply:
Yes, I tried it with an existing project